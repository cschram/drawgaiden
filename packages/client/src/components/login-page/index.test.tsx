import React from 'react';
import renderer from 'react-test-renderer';
import { LoginPage } from './index';
import testData from '../../../test/data/canvas';

test('<App/>', () => {
    const props = {
        loggingIn: false,
        loginError: 'Unknown error.',
        login: (username, redirect) => {},
        redirect: '/canvas/12345'
    };
    const tree = renderer.create(<LoginPage {...props}/>).toJSON();
    expect(tree).toMatchSnapshot();
});