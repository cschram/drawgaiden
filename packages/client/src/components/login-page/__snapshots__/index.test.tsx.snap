// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App/> 1`] = `
<div
  className="login-page"
>
  <form
    className="login-form"
    onSubmit={[Function]}
  >
    <img
      alt="Draw Gaiden"
      src={Object {}}
    />
    <fieldset>
      <span
        className="error"
      >
        Unknown error.
      </span>
      <input
        placeholder="Enter user name"
        type="text"
      />
      <button
        onClick={[Function]}
      >
        <span>
          Login
        </span>
      </button>
      <p>
        Having issues? Check our 
        <a
          href="https://github.com/drawgaiden/drawgaiden/issues"
          target="_blank"
        >
          bug tracker
        </a>
        .
      </p>
    </fieldset>
  </form>
</div>
`;
